### PR TITLE
Update three-to-ammo dependency to new c-frame fork

### PR DESCRIPTION
**Pull request by Elettrotecnica**
See https://github.com/c-frame/aframe-physics-system/issues/59

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "ammo-debug-drawer": "^0.1.0",
     "cannon-es": "^0.9.1",
     "patch-package": "^6.5.0",
-    "three-to-ammo": "^1.0.1",
+    "@c-frame/three-to-ammo": "^1.0.3",
```
Comment (diarmidmackenzie) [inline package.json:48]: I just published 1.0.2.  Not sure what 1.0.3 is?

Comment (diarmidmackenzie) [reply package.json:48]: OK - just seen https://github.com/c-frame/three-to-ammo/pull/5 & now understand.  I will merge & pubish that and then come back to this.

**Comment by diarmidmackenzie** [comment]
To get this to build, I had to update references in `ammo-body.js` and `ammo-shape.js` to:

```
const threeToAmmo = require("@c-frame/three-to-ammo");
```

Not sure whether that's the best fix, or if there's some way to alias three-to-ammo to @c-frame/three-to-ammo in package.json, which might be a more elegant solution?

**Comment by Elettrotecnica** [comment]
Sorry for not checking thoroughly, should be fine now, I have used an alias so that the logical dependency does not change

**Comment by diarmidmackenzie** [comment]
Thanks for this.  Do you need me to publish an update to npm?

**Comment by Elettrotecnica** [comment]
> Thanks for this. Do you need me to publish an update to npm?

three-to-ammo 1.0.3 is the latest and it seems you have published it. I think we are all set! :-)

Thanks a lot!

**Comment by Elettrotecnica** [comment]
Concerning this repo, personally I do not use this from cdn, but maybe a release wouldn't be bad.

I could look into it if you prefer.

**Pull request merged**